### ISSUE TITLE
Learning curve isn't discontinuous when i continued training it on a pre-trained model

### ISSUE THREAD

**Issue by powder21** [openai/pixel-cnn#32]
I stopped training after 60 epoch. Then i restored the model and run the code again, but the NLL at the beginning of the 2nd training was much larger than the value before i stopped training. In other words, the model can't restore the status before. Is there some temporary variables losing in the model or something else?

**Comment by powder21** [comment]
I forgot the learning rate decay.
